server/src/test/java/org/elasticsearch/discovery/DiscoveryDisruptionIT.java

```diff
@@ -54,13 +53,12 @@
 /**
  * Tests for discovery during disruptions.
  */
-@ESIntegTestCase.ClusterScope(scope = ESIntegTestCase.Scope.TEST, numDataNodes = 0, transportClientRatio = 0, autoMinMasterNodes = false)
```

Comment:
I think this shouldn't be removed, although the `autoMinMasterNodes = false` can go.